es_loader/es_loader.py

```diff
@@ -41,12 +72,49 @@ def _create_index(self):
         self.es.indices.create(self.index_name)
         logger.debug('Index created')
 
+    def _get_job(self):
+        """Get job from ScrapingHubClient istance."""
+        self.job = self.sc.get_job(self.job_id)
+
+    def _get_job_metadata(self):
+        """Get job metadata."""
+        self.metadata = self.job.metadata.list()
+
+    def _get_scraped_items_count(self):
+        """Get scraped items count from job metadata.
+
+        The count is needed to make log messages more verbose
+        and to calculate batches size"""
+        self._get_job_metadata()
+        for data in self.metadata:
+            if data[0] == 'scrapystats':
```

Comment:
Do we need `else` here? What happens if we don't find `scrapystas` in data? Maybe just raise an exception at the end of the method if items_count is still not known.